### ISSUE TITLE
No children props in the latest npm version

### ISSUE THREAD

**Issue by ahronsilv** [chakrahq/react-native-side-menu#4]
Hello. From what I see on github source you have children prop in your index.d.ts. However, this version is not available in NPM. Are you planning on uploading this version soon?

Thank you

**Comment by Raamzeez** [comment]
Yes please update this, it is an important change and the compiler can be happy
